SpotifyUserLogin/src/main/java/main/Main.java

```diff
@@ -1,82 +1,135 @@
 package main;
 
+import api.SpotifyInteractor;
 import entities.UserProfile;
-import entities.Playlist;
-import entities.Song;
-import View.DashboardView;
-import View.ProfileRateAndCommentView;
-import View.ProfileView;
-import View.MainMenuView;
-import View.PlaylistView;
+import interface_adapter.editpreferences.EditPreferencesController;
+import interface_adapter.editpreferences.EditPreferencesPresenter;
+import interface_adapter.editpreferences.EditPreferencesState;
+import Use_case.Editing.EditPreferencesUseCase;
+import Use_case.Editing.EditPreferencesResponse;
 
 import javax.swing.*;
 import java.awt.*;
-import java.util.Arrays;
-import java.util.List;
+import java.util.ArrayList;
 
 public class Main {
     public static void main(String[] args) {
```

Comment:
Initializations and view instantiation should eventually be moved into a builder so that it's more readable, and easier to add on extra parts to the application